### ISSUE TITLE
_NOT_PARSED_ in embedded property

### ISSUE THREAD

**Issue by vitorenesduarte** [orientechnologies/orientdb#3613]
__2.0.2__
In studio:
```sql
create class Product
create property Product.data EMBEDDED
insert into Product set data = {currency:'EUR'}
```
the last returns:
![screen shot 2015-02-17 at 15 08 44](https://cloud.githubusercontent.com/assets/4520516/6230303/7ea5a6fa-b6b6-11e4-9d3a-9667077e83a5.png)

**Comment by vitorenesduarte** [comment]
Same with:
```sql
insert into Product set data = {currency:EUR}
```

This also occurs:

```sql
insert into Product set data = 1234 # gives an error (ok)
insert into Product set data = potato # inserts a document with property data = null 
```

**Comment by rmuller** [comment]
You are not using valid JSON, should be:
insert into p set data = {'currency': 'EUR'} (note 'currency')

Also note the clause CONTENT support the insertion of JSON data.

Note this is an issue system, not a Q&A site (see http://stackoverflow.com/questions/tagged/orient-db)

**Comment by vitorenesduarte** [comment]
Hi @rmuller. 
I'll explain. OrientDB throws an exception when I try to:
```sql
insert into Product set data = 1234
```
So, why doesn't that happen in the other cases?

If you still think this is not an issue, I'll close it.

**Comment by rmuller** [comment]
My comment concerns the "_NOT_PARSED_" exception and why it occurs. `1234` is a literal value, which cannot be parsed as EMBEDED. Same for 

````
insert into Product set data = 'potato'
````

Your example `insert into Product set data = potato` potato is not a literal value but a field, which does not exits and is considered `null`. I do not know if this is the expected behaviour, but for me it makes sense.
 Hope this is helpful for you.

**Comment by vitorenesduarte** [comment]
You're right, thanks.
About "NOT_PARSED", I think it should throw an exception like the others, instead of inserting the record. Don't know if you agree.

**Comment by rmuller** [comment]
That makes sense. But i am not an orientdb maintainer, so have no strong opinion about this.

**Comment by luigidellaquila** [comment]
Hi guys, 

this issue is already fixed in 2.1

I'm closing this issue, please reopen if you still experience related problems

Thanks

Luigi
